homeassistant/components/light/lifx.py

```diff
@@ -227,6 +227,25 @@ def turn_on(self, **kwargs):
         self._liffylights.set_color(self._ip, hue, saturation,
                                     brightness, kelvin, fade)
 
+    def set_brightness(self, **kwargs):
```

Comment:
Calling the service will call the service call handler `handle_light_service`. This one will call 

```python
             for light in target_lights:
                 getattr(light, service_fun)(**params)
```

And expanding a dictionary to a function will map the keys to the arguments just fine:

![screen shot 2016-06-25 at 24 16 08](https://cloud.githubusercontent.com/assets/1444314/16355394/3dce7790-3a6a-11e6-8c0b-3c095bb7386a.png)